### PR TITLE
dependency: replace js-sha3 to noble hashes

### PR DESCRIPTION
**Pull request by joshstevens19**
replace js-sha3 to noble hashes as that is a more mature and actively funded repo

tests all pass 

requested on issue > https://github.com/joshstevens19/ethereum-bloom-filters/issues/34

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import sha3 = require('js-sha3');
+import { keccak_256 } from '@noble/hashes/sha3';
 
 /**
  * Keccak256 hash
  * @param data The data
  */
 export function keccak256(data: string | ArrayLike<number>): string {
-  return '0x' + sha3.keccak_256(toByteArray(data));
+  return bytesToHex(keccak_256(toByteArray(data)));
```
Comment (paulmillr) [inline src/utils.ts:8]: `'0x' + bytesToHex(`

**Comment by paulmillr** [comment]
fix 0x and looks good!

File: src/utils.ts (hunk 1 above)
Comment (joshstevens19) [reply src/utils.ts:8]: Bytes to hex already does that

Comment (joshstevens19) [reply src/utils.ts:8]: https://github.com/joshstevens19/ethereum-bloom-filters/blob/163da5ea767e95362196264b68956b49587a8126/src/utils.ts#L37

Comment (paulmillr) [reply src/utils.ts:8]: ok

**Comment by joshstevens19** [comment]
> fix 0x and looks good!

Cool replied to that comment it does do it lower down, if looks good will merge and deploy tomorrow

**Pull request merged**